docs/INSTALL.rst

```diff
@@ -70,7 +70,7 @@ for python2 and python3 respectively. These packages are tested, and should work
 with teuthology. But if you want to bump up the versions of them, please use the 
 following command to update these files. For python3 ::
 
-  pip-compile -qo- | sed '/^-e / d' > requirements.txt
+  echo "-e .[orchestra,test]" | pip-compile - -qo- | sed '/^-e / d' > requirements.txt
```

Comment:
Instead of teaching users to read the docs, we could create a shorthand script for updating requirements, like `/update_requirements` or `/pip-compile`:
```
echo "-e .[orchestra,test]" | pip-compile - -qo- | sed '/^-e / d' > requirements.txt
```

Reply:
More advanced script which can bypass arguments to pip-compile:
```
export DIR=$(cd $(dirname $BASH_SOURCE); echo $PWD)
cd $DIR
echo "-e .[orchestra,test]" | pip-compile $@ - -qo- | sed '/^-e / d' > requirements.txt

```

Reply:
again, then he/she needs to read read document to understand that `update_requirements` or `pip-compile` should be used for generating `requirements.txt`. due to the activity level and popularity of this project, i don't think it's worthwhile.

Reply:
I agree, but this makes much easier to copy-paste pip-compile, I found myself boring to got to docs and search for the right command to run, instead it is much easier to recall there is a script in the root of the repo.

Reply:
@kshtsk updated and repushed.